portablemc/cli/__init__.py

```diff
@@ -273,14 +273,19 @@ def cmd_search_handler(ns: SearchNs, kind: str, table: OutputTable):
             if search is None or search in alias:
                 table.add(alias, version)
 
-    elif kind in ("fabric", "quilt"):
+    elif kind in ("fabric", "quilt", "legacyfabric"):
 
-        from ..fabric import FABRIC_API, QUILT_API
+        from ..fabric import FABRIC_API, QUILT_API, LEGACYFABRIC_API
 
         table.add(_("search.loader_version"))
         table.separator()
 
-        api = FABRIC_API if kind == "fabric" else QUILT_API
+        if kind == "fabric":
+            api = FABRIC_API()
+        elif kind == "quilt":
+            api = QUILT_API()
+        else:
```

Comment:
Now that we have many fabric APIs, could you fully specify the name with a "elif", and raise a ValueError in the else. Just in case, may help us in the future.

Reply:
I figured that since there is a `raise ValueError()` in L294 right after this block, that it would be redundant to do it again.